writeengine/bulk/we_bulkload.cpp

```diff
@@ -785,7 +785,7 @@ int BulkLoad::preProcess( Job& job, int tableNo,
                                  job.jobTableList[tableNo].tblName,
                                  lbid,
                                  oldHwm, hwm,
-                                 bSkippedToNewExtent, false) );
+                                 bSkippedToNewExtent, bSkippedToNewExtent || oldHwm < 1) );
```

Comment:
Just wondering, is this for situation when we have more than 1 extent per segment file?